### PR TITLE
Add tidy check to the test suite

### PR DESCRIPTION
**Pull request by toddr**
By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: t/components-AbsoluteSymlink.t
Comment (toddr) [inline t/components-AbsoluteSymlink.t:1]: Tidy only changes.

File: t/blocker-Repositories.t
Comment (toddr) [inline t/blocker-Repositories.t:1]: Tidy only changes.

File: t/blocker-Python.t
Comment (toddr) [inline t/blocker-Python.t:1]: Tidy only changes.

File: t/blocker-DiskSpace.t
Comment (toddr) [inline t/blocker-DiskSpace.t:1]: Tidy only changes.

File: script/elevate-cpanel.PL
Comment (toddr) [inline script/elevate-cpanel.PL:1]: Tidy only changes.

**Review by timmullin — APPROVED** [review]

**Pull request merged**